src/nupic/algorithms/Cells4.cpp

```diff
@@ -1336,37 +1326,7 @@ void Cells4::compute(Real* input, Real* output, bool doInference, bool doLearnin
   // invalidating our indexes.
   memset(output, 0, _nCells * sizeof(output[0])); // most output is zero
 #if SOME_STATES_NOT_INDEXED
-#ifdef NTA_PLATFORM_darwin86
```

Comment:
@scottpurdy it's speed-only change, I don't have hw to measure, but i thought it was a removal of a dead code only - I dont think we are now able to build nupic on 32b OSX, which was with CoreDuo CPUs ~2006.